utils/auction.ts

```diff
@@ -263,6 +264,46 @@ export const verifyMessageWithTime = (data: MessageWithTimeSignature, signature:
   return address.toLowerCase() === addr.toLowerCase()
 }
 
+export const validateOrder = async (order: Order, auction: Auction) => {
+  let isValidOrder = true;
+  let response = '';
+
+  if (auction.minSize == 0 || auction.auctionEnd == 0) {
```

Comment:
We should not throw this error. If the auction is not started, we allow user create bid which will be filtered by King crab